app/controllers/foreman_puppet/api/v2/base_controller.rb

```diff
@@ -3,7 +3,8 @@ module Api
     module V2
       class BaseController < ::Api::V2::BaseController
         resource_description do
-          api_version '2'
+          resource_id 'foreman_puppet_base'
```

Comment:
Where is the id of the resource used?

Reply:
It is used by Apipie to distinguish between the classes and get the documentation correctly, as far as I can tell.

Reply:
I should change the resource_id of the `BaseController` and the one of the `LookupsCommonController` to have a `pp_` prefix such that all of the resource_ids are consistent.